mros2_msgs/msg/QoS.msg

```diff
@@ -0,0 +1,3 @@
+string objective_id
+string selected_mode
```

Comment:
@jginesclavero  Just to confirm, you're checking when the selected_mode changes?

Reply:
also, if `selected_mode` is left empty in the request, the reasoner will select the best mode